sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamingQueryStatusSuite.scala

```diff
@@ -84,6 +88,20 @@ class StreamingQueryStatusSuite extends SparkFunSuite {
       StreamingQueryStatus.testStatus.prettyJson ===
         """
           |{
+          |  "name" : "query",
+          |  "id" : 1,
```

Comment:
I wonder if we should make this a `String`, both here and in the type-safe API.  The reasoning is, its already auto generated and I think its likely we will want to turn this into a UUID (maybe we should even just do that).  Thoughts?